File: Data Structures/Graph/Adjacency Matrix/graph.py
n = 5

graph = [ [0 for i in range(n)]for j in range(n) ]

A, B, C, D, E = 0, 1, 2, 3, 4

def addEdge(nodeA, nodeB):
    graph[nodeA][nodeB] = 1

for row in graph:
    print(row)

addEdge(A, B)
addEdge(B, A)
addEdge(C, B)
addEdge(C, D)
addEdge(D, A)
addEdge(D, B)

tab = 'A B C D E'.split(' ')
print()
print(tab)
for i, row in enumerate(graph):
    print(f'{tab[i]} {row}')

File: Data Structures/Graph/Adjacency Matrix/graph.c
#include <stdio.h>

enum node { A, B, C, D, E, F };

#define n 6
int G[n][n];

void zeroFill(int g[n][n])
{
	for(int i = 0; i<n; i++)
	{
		for(int j = 0; j<n; j++)
		{
			g[i][j] = 0;
		}
	}
}

void addEdge(int nodeA, int nodeB)
{
	G[nodeA][nodeB] = 1;
}

void show( char* name, int G[n][n] )
{
	int i, j;
	printf("Graf: %s\n", name);

	for(i = 0; i < n; i++)
	{
		for(j = 0; j < n; j++)
		{
			printf("%i ", G[i][j]);
			if(j == n-1) printf("\n");
		}
	}
}

int main(void)
{
	zeroFill(G);

	addEdge(A, B);
	addEdge(B, C);
	addEdge(B, D);
	addEdge(C, C);
	addEdge(C, E);
	addEdge(D, E);
	addEdge(D, F);
	addEdge(E, F);

	show("G", G);
}
